source/blog/2017-01-09-building-static-sites-with-middleman.html.markdown

```diff
@@ -0,0 +1,89 @@
+---
+title: Building Static Sites with Middleman
+date: 2017-01-09 19:58:57 UTC
+background_image: /img/blog/building-static-sites-with-middleman/background.jpg
+background_image_source: https://twitter.com/CloudyConway/status/817193041161232385
+---
+
+I recently launched this [Peer Lab website](http://peerlab.community) to list the peer labs around the world and to provide instructions on starting one. It's a small site with no need for a backend, so I decided to host it as static HTML. However, it was important that adding new labs would be easy for outside contributors. Since I had used the tool before to build a few static websites – including this blog – I turned to [Middleman](https://middlemanapp.com).
```

Comment:
to list the peer labs around the world and provide instructions to start your own.

However, it was important for it to be easy for outside contributors to add new labs.